### PR TITLE
update-logo

### PR DESCRIPTION
**Pull request by tihuan**
Deprecate Corpora 💀 And long live cellxgene 👼 🎉 

Screen Shot 2020-08-25 at 2 34 45 PM

### CODE REVIEW

File: frontend/src/common/images/logo.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
...
... 7.9 5.4 8.8 5.8 91.8l.4 76.7 28.7-.2 28.7-.3.3-69.5-.8-85c-2.3-34.1-12.8-55.7-31.8-65.4-10.5-5.4-28.8-7-41.7-3.7zm180 .1c-16.2 3.3-27.3 8.6-37.8 18.1-13 11.8-21.2 26.2-26.5 46.9-3.9 15.1-4.7 23.6-4.7 51.5.1 24 .3 28.2 2.3 37.7 5.5 26.6 16.1 44 34.9 57.4 23.2 16.7 61.9 21.1 92.2 10.7 16.4-5.6 28.2-13.2 37.9-24.4l2.2-2.6-10.6-16.4-11.4-17.2c-.4-.5-4.1 1.6-8.1 4.7-12.7 9.7-28.3 14.4-44.4 13.3-16-1-25.2-6.7-31.1-19.3-3.3-7.1-7.1-24.1-6-27 .5-1.2 9.6-1.4 57.7-1.4h57.1l-.4-26.8c-.6-32.6-2-41.4-10-61.2-7.9-19.6-23-34.3-41.7-40.6-16.2-5.4-35.4-6.7-51.6-3.4zm32.1 49.1c8.6 4.1 13.3 14.7 14.1 31.6l.6 11.3H2016v-4.2c0-6.7 3-20.9 5.6-26.3 5-10.8 13.7-15.9 25.7-15.1 4.2.3 8.6 1.4 11.3 2.7zm-879.2 5.6l-9.8 10.5c-1 .7-.9 1 6.9 10.4l13.8 16.8 15 18.2c6.4 7.8 11.7 14.5 11.7 14.9 0 .3-6 7.8-13.2 16.7l-33.8 42.2c.1.6 4.3 5.6 9.5 11.1 8.2 8.8 9.6 9.8 10.8 8.5 1.6-1.7 26.6-32 37.6-45.8 4.1-5 7.8-9.2 8.3-9.2.4 0 4 4 8 8.8l23.2 28c8.7 10.6 16.2 19.1 16.7 19 1.1-.4 18.9-19.8 18.9-20.6 0-.4-10.4-13.3-23.1-28.7l-23.9-29c-.6-.7 8.5-12.4 23.4-30.5l24.4-29.5-2.9-3.5c-5.4-6.6-16.7-18.5-17.1-18-.3.3-11 13.3-23.8 28.9l-23.4 28.4-3.4-3.9c-4.1-4.7-36.5-44.1-40.7-49.5-1.6-2.1-3.3-3.8-3.6-3.9-.4 0-4.6 4.4-9.5 9.7z"/>
```
Comment (seve) [inline frontend/src/common/images/logo.svg:3]: Do we have defined use cases for black vs pink versions of the logo?

Comment (tihuan) [reply frontend/src/common/images/logo.svg:3]: I don't think so! @pdugan20 is indifferent between either, and so I picked black for the homepage to match the cellxgene page!

What's your thought on this 😄 ?

Comment (seve) [reply frontend/src/common/images/logo.svg:3]: No thoughts, just curious on if we have a standard I should be following in the future.  For now, it seems that the only use cases for the colored logo are print and publishing (stickers, features in papers/posters, our doc website)

Comment (tihuan) [reply frontend/src/common/images/logo.svg:3]: Ah awesome context to keep in mind! Thanks for sharing that, @seve 🙏 👌

**Pull request merged**